src/components/SkipTo.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const Div = styled.div`
+    background-color: ${(props) => props.theme.colors.primary};
+`
+
+const Anchor = styled.a`
+    line-height: 2rem;
+    position: absolute;
+    top: -3rem;
+    margin-left: 0.5rem;
+    color: ${(props) => props.theme.colors.background};
+    text-decoration: none;
+
+    &:focus {
+        position: static;
+    }
+`
+
+const SkipTo = ({ href_id }) => {
```

Comment:
Calling this component a SkipLink makes more semantic sense to me

```suggestion
const SkipLink = ({ href_id }) => {
```